platform/android/java/app/config.gradle

```diff
@@ -289,6 +289,14 @@ ext.getDebugKeyAlias = { ->
     return keyAlias
 }
 
+ext.getDebugAliasPassword={->
+    String keyAliasPassword = project.hasProperty("debug_keystore_alias_password") ? project.property("debug_keystore_alias_password") : ""
+    if(keyAliasPassword==null||keyAliasPassword.isEmpty()){
```

Comment:
I'm not sure if we have style guide for Gradle files, but I think it's better to use the same format as codes around this.

Reply:
already edited.

Reply:
already edited.